### PR TITLE
Split stylesheet based on page and optimize css 

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Move page related styles in respective stylesheet
2. Move page related stylesheet from app.scss
3. Include page styles in the template
3. Remove unused styles from mui and reflex
4. Update profile banner design

### CODE REVIEW

File: funnel/assets/js/utils/vue_util.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const faSvg = Vue.component('faicon', {
...
...a class="mui--text-body2 js-copy-link" href="javascript:void(0)" data-ga="Copy link"><faicon :icon="'copy'" :baseline=true :css_class="'mui--text-light fa-icon--right-margin'"></faicon>Copy link<span class="js-copy-url" aria-hidden="true">{{ url }}</span></a></li><li><a class="mui--text-body2" :href="emailUrl" data-ga="Share via Email"><faicon :icon="'envelope'" :baseline=true :css_class="'mui--text-light fa-icon--right-margin'"></faicon>Email</a></li><li><a class="mui--text-body2" target="_blank" rel="noopener" :href="twitterUrl" :data-url="url" data-via="Hasgeek" :data-text="title" data-ga="Tweet"><faicon :icon="'twitter-square'" :baseline=true :css_class="'mui--text-light fa-icon--right-margin'"></faicon>Twitter</a></li><li><a class="mui--text-body2" target="_blank" rel="noopener" :href="facebookUrl" :data-href="url" data-ga="Share on facebook"><faicon :icon="'facebook-square'" :baseline=true :css_class="'mui--text-light fa-icon--right-margin'"></faicon>Facebook</a></li><li><a class="mui--text-body2" :href="linkedinUrl" data-ga="Share about on linkedin"><faicon :icon="'linkedin'" :baseline=true :css_class="'mui--text-light fa-icon--right-margin'"></faicon>Linkedin</a></li><ul>`,
```
Comment (jace) [inline funnel/assets/js/utils/vue_util.js:68]: gettext missing here.

File: funnel/templates/scan_contact.html.jinja2 [hunk 2]
```diff
@@ -5,11 +5,12 @@
 {% block title %}{% trans %}Scan badge{% endtrans %}{% endblock %}
 
 {% block pageheaders %}
-<style>
-  html, body {
-    overflow: hidden;
-  }
-</style>
+  <style>
```
Comment (jace) [inline funnel/templates/scan_contact.html.jinja2:8]: Doesn't this need a type declaration?

File: funnel/templates/update_logo_modal.html.jinja2 [hunk 3]
```diff
@@ -1,6 +1,8 @@
 {% from "macros.html.jinja2" import faicon %}
 {% from "forms.html.jinja2" import renderfield %}
 
+<link rel="stylesheet" type="text/css" href="{{ built_asset('css/imgee-modal.scss') }}" />
```
Comment (jace) [inline funnel/templates/update_logo_modal.html.jinja2:4]: Shouldn't `<link>` tags be in the `<head>` section of HTML?

File: funnel/templates/venues.html.jinja2 [hunk 4]
```diff
@@ -20,7 +20,7 @@
             <ol class="mui-list--aligned mui-list--border">
               {%- for venue in primary_venue_form.venue.query %}
                 <li class="mui--text-subhead">
-                  <input id="venue-{{ loop.index }}" name="{{ primary_venue_form.venue.name }}" value="{{ venue.uuid_b58 }}" type="radio" {%- if venue == project.primary_venue %}disabled{% endif %} class="form-inline-input"><label for="venue-{{ loop.index }}" class="form-inline-label" data-cy="{{ venue.title }}">{{ venue.title }} {%- if venue == project.primary_venue %}<em> (primary) </em>{% endif %}</label>
+                  <input id="venue-{{ loop.index }}" name="{{ primary_venue_form.venue.name }}" value="{{ venue.uuid_b58 }}" type="radio" {%- if venue == project.primary_venue %}disabled{% endif %} class="margin-right"><label for="venue-{{ loop.index }}" class="form-inline-label" data-cy="{{ venue.title }}">{{ venue.title }} {%- if venue == project.primary_venue %}<em> (primary) </em>{% endif %}</label>
```
Comment (jace) [inline funnel/templates/venues.html.jinja2:23]: `trans` tag missing for "(primary)" annotation.

**Review by jace — CHANGES_REQUESTED** [review]

File: funnel/templates/update_logo_modal.html.jinja2 (hunk 3 above)
Comment (vidya-ram) [reply funnel/templates/update_logo_modal.html.jinja2:4]: This is a partial template opened in a modal via ajax

**Pull request merged**